packages/functions/src/utils/DbAppointmentUtils.ts

```diff
@@ -120,6 +121,25 @@ export function toBookedAppointmentSync(
   }
 }
 
+export async function getAppointmentById(
+  appointmentId: string
+): Promise<DbAppointment> {
```

Comment:
If this doesn't throw, it should return `DbAppointment | undefined`

Reply:
Done

Reply:
None of the functions are built to handle undefined,
I removed the unsafe method

Reply:
Great